del_items_from_list_ex.py

```diff
@@ -7,16 +7,40 @@
 #
 #   12-OCT-18 : PTR XXXX, Created, Dodd M. Thomas
 #
+#   29-NOV-18 : PTR XXXX, Modified, Logan Thomas
```

Comment:
Typically wouldn't use "Modified" here, it would be one line description describing mod